client/data/settings/test/hooks.js

```diff
@@ -267,12 +256,12 @@ describe( 'Settings hooks tests', () => {
 				updateDigitalWalletsLocations,
 			] = useDigitalWalletsLocations();
 
-			updateDigitalWalletsLocations( locationsAfterUpdated );
+			updateDigitalWalletsLocations( locationsAfterUpdate );
 
-			expect( digitalWalletsLocations ).toEqual( locationsBeforeUpdated );
+			expect( digitalWalletsLocations ).toEqual( locationsBeforeUpdate );
 			expect(
 				actions.updateDigitalWalletsLocations
-			).toHaveBeenCalledWith( locationsAfterUpdated );
+			).toHaveBeenCalledWith( [ 'cart' ] );
```

Comment:
When writing tests, I like to use as few constants as possible. Not using constants (also when importing action types) gives higher confidence that the test are working as expected.
It looks like I didn't change `locationsBeforeUpdate`, tho - changing this too just so that I can edit it in GH #lazyness 😄